infra/docker-compose/dev/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
 
   # Kubernetes control plane
   server:
-    image: "rancher/k3s:v1.19.4-k3s1"
+    image: "rancher/k3s:v1.20.12-k3s1"
```

Comment:
>Since k3s are sensitive to the Linux VM that Docker runs, here's a working and tested version of docker. I noticed k3s 1.19 breaks on my setup.

Curious - is there some official doc with a compatibility map ?

Reply:
I didn't dig deep into this issue but 1.19 used to work on my machine also.